packages/netlify-cms-backend-bitbucket/src/API.ts

```diff
@@ -653,8 +653,8 @@ export default class API {
       params: {
         pagelen: 50,
         q: oneLine`
-        source.repository.full_name = "${this.repo}" 
-        AND state = "${BitBucketPullRequestState.OPEN}" 
+        source.repository.full_name = "${this.repo}"
+        AND state = "${BitBucketPullRequestState.OPEN}"
```

Comment:
Hm... I guess this will break. We need to enforce spaces here somehow.

Reply:
Let me look into that. I'm mostly concerned that this doesn't happen when I run `yarn format`

Reply:
Hm... I does happen when I run ```yarn format```

Reply:
Oh, no, I was wrong. This is something my IDE is doing on save. Weird.

Reply:
That's very odd as we also run prettier in CI on Windows, MacOS and Ubuntu and it passes.

Maybe something specific to your machine?

Regardless, they are redundant so I'll delete them in another PR and fix the recorded data [here](https://github.com/netlify/netlify-cms/tree/e4a29d5991f648b98b7df9f1a9a8c6448560851b/cypress/fixtures)

Reply:
Well actually looks like tests are passing so it seems `oneLine` trims those so we don't need to update any recorded data.

Reply:
> Oh, no, I was wrong. This is something my IDE (WebStorm) is doing on save. Weird.

Just noticing this, I still think we can remove those